dxr/static_unhashed/js/context_menu.js

```diff
@@ -168,7 +166,12 @@ $(function() {
                 menuItems = menuItems.concat(currentNodeData);
             }
 
-            contextMenu.menuItems = menuItems;
+            if (menuItems.length === 0) {
+                // Nothing to display.
+                return;
+            }
+
+            var contextMenu = { menuItems: menuItems };
```

Comment:
LGTM. If you want, you could even merge this into the next line; I don't really see a clarity win for this one-use `contextMenu` symbol given that it's being passed to `setContextMenu()`.